tailwind.config.js

```diff
@@ -48,6 +48,7 @@ export const platform = [
 ];
 
 /** @type {import("tailwindcss").Config} */
+// eslint-disable-next-line import/no-anonymous-default-export
 export default {
   // mode: "jit",
```

Comment:
This looks like commented code, I'd probably remove it. Other than that LGTM.

Reply:
Without it, the code editor displays a multitude of yellow squiggles, making it difficult to identify and update specific areas. Let me know what do you think?

![image](https://github.com/SSWConsulting/SSW.Website/assets/71385247/10798466-c52e-4026-9e38-a40dbbb9ccff)

Reply:
I was referring to the `// mode: jit,` line. The linting configuration is fine :).

Reply:
oh right! Actually with comment this line is highlighted
![image](https://github.com/SSWConsulting/SSW.Website/assets/71385247/8c79d3e0-68d5-4b81-8ae4-934c28f0b50c)

Reply:
the commented code might not have been a file change so when I clicked it added the comment to the highlighted change instead

Reply:
done